BB27EBFFFE89A31A.json

```diff
@@ -0,0 +1,20 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "BB27EBFFFE89A31A",
+    "servers": [
+      {
+        "server_address": "eu1.thethings.network",
```

Comment:
This is incorrect. The addres of the EU server is `eu1.cloud.thethings.network`, please change and then we can merge

Reply:
all done i hope

Reply:
👍